1-bootstrap/tf_image.tf

```
@@ -35,27 +35,38 @@ resource "google_service_account" "builder" {
   account_id = "tf-builder"
 }
 
-resource "google_storage_bucket" "build_logs" {
-  name                        = "cb-tf-builder-logs-${var.project_id}"
-  project                     = var.project_id
-  uniform_bucket_level_access = true
-  force_destroy               = var.bucket_force_destroy
-  location                    = var.location
-  logging {
-    log_bucket        = var.logging_bucket
-    log_object_prefix = "cb-ai-${var.project_id}"
-  }
-
-  versioning {
-    enabled = true
-  }
+module "build_logs" {
+  source  = "terraform-google-modules/cloud-storage/google//modules/simple_bucket"
+  version = "10.0.2"
```

Would this make more sense?

```suggestion
  version = "~> 10.0"
```